example/rust-example.md

```diff
@@ -0,0 +1,75 @@
+# Prerequisites
+
+There need rust toolchain in your local environment, if you haven't set it up, you can refer [here](https://www.rust-lang.org/tools/install) to figure it out.
+
+# Dependencies
+
+put the content below to Cargo.toml
+
+```toml
+[package]
+name = "opendal-example-rust"
+version = "0.1.0"
+edition = "2023"
+
+[dependencies]
+opendal = { version = "0.34.0", features = ["services-webdav", "services-gdrive"] }
+tokio = { version = "1.27", features = ["full"] }
+anyhow = { version = "1.0.30", features = ["std"] }
+```
+
+# Usage
+
+Here shows how to use OpenDAL to manipulate file on WebDAV server and Google Drive.
+
+```rust
+use opendal::Operator;
+use opendal::Result;
+use opendal::services::Gdrive;
+use opendal::services::Webdav;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+
+    // create a webdav builder and configure it
+    let mut webdav_builder = Webdav::default();
+    webdav_builder
```

Comment:
webdav is not recommanded to be used as the first example (becuase it requres user to have a webdav service setup).

Reply:
haha. I chose webdav and gdrive for example cause they are the only service that I have used. so what services do you think are suitable as the good first example?

Reply:
In rust-day-1, I prefer to use `fs` instead.

Reply:
make sense